schedule/supportserver_generator.yaml

```diff
@@ -0,0 +1,36 @@
+---
+name: supportserver_generator
```

Comment:
Is it third approach where we conduct full installation? I would say that one would be enough and `schedule/supportserver_generator_from_hdd.yaml` is more efficient in that sense.

Reply:
i mainly worked with full installation. i agree that `schedule/supportserver_generator_from_hdd.yaml` would be ideal but i included it just as documentation of the research but i have not robust research around it. for instance, i did not see it run successful with textmode. and i have not tried to make it work as i was spending time with `schedule/supportserver_generator.yaml`. 
i know that this is not a satisfied respond at all and it troubles too because of the time spent so far in combination with the results. Do you want to talk about it in details?

Reply:
My main concern is that realistically we unlikely to maintain all 3 solutions you have provided. As per you comment below we also don't have perfect solution as one using `hdd` from `create_hdd` job doesn't work, and `schedule/supportserver_generator.yaml` is just slower. So I would focus on one solution.

Reply:
i did not focus on the faster version as i was thinking that we do not generate a new image very often and it would not be so important. But i can take a look

Reply:
As per our current experience, if we don't run that job regularly, it won't work next time we need it. So I thought of running generation job regularly in Development job group, for instance.